.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/213629092-3d6883f6-4a8c-4599-be00-e6d7e61c8214.png)

- [ ] You don't have a descriptive title for your pull request. :fearful: KIndly change Dev to something like ** Awesome books refactored with ES6** to make your pull request professional :+1:

Reply:
Thank You @HAMMAS-SALEEM , i will make the changes and request a new review